packages/story-editor/src/app/story/storyProvider.tsx

```diff
@@ -35,8 +36,17 @@ import useStoryReducer from './useStoryReducer';
 import useAutoSave from './actions/useAutoSave';
 import { StoryTriggersProvider } from './storyTriggers';
 import useLocalAutoSave from './actions/useLocalAutoSave';
+import type {AddPageAction} from "../../types/storyProvider";
 
-function StoryProvider({ storyId, initialEdits, children }) {
+interface ProviderProps {
+  storyId: number;
+  initialEdits?: {
+    [index: string]: unknown;
```

Comment:
unknown?

Reply:
Not typed yet properly. This PR is still WIP and mostly not done, looks like Zenhub moved it to Review somehow, it's supposed to be In Progress, moved it there.